dev-games/openscenegraph-openmw/files/openscenegraph-3.6.5-use_boost_asio.patch

```diff
@@ -0,0 +1,274 @@
+diff -ruN OpenSceneGraph-OpenSceneGraph-3.6.5/CMakeLists.txt osg-my/CMakeLists.txt
+--- OpenSceneGraph-OpenSceneGraph-3.6.5/CMakeLists.txt	2020-01-31 12:03:07.000000000 +0100
++++ osg-my/CMakeLists.txt	2020-09-26 18:45:41.191099844 +0200
```

Comment:
please clean up these patches

Reply:
This was copied as is from openscenegraph ebuild.

I'll clean up there first in a separate commit.